### PR TITLE
[TR] Upgrade phpcs build strategy to use php-cs-fixer

### PR DESCRIPTION
**Pull request by grena**
- Specs: :white_check_mark: 
- Behat ORM: :white_check_mark: 
- Behat Mongo: :white_check_mark:

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Form/Type/MassEditAction/AddToGroupsType.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
         $resolver->setDefaults(
             [
                 'data_class' => 'Pim\\Bundle\\EnrichBundle\\MassEditAction\\Operation\\AddToGroups',
-                'groups' => array(),
+                'groups'     => array(),
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Form/Type/MassEditAction/AddToGroupsType.php]: [] ?

File: src/Pim/Bundle/EnrichBundle/Form/Type/MetricType.php [hunk 2]
```diff
@@ -40,7 +40,7 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
         $resolver->setDefaults(
             array(
-                'data_class' => 'Pim\Bundle\CatalogBundle\Model\Metric',
+                'data_class'   => 'Pim\Bundle\CatalogBundle\Model\Metric',
                 'units'        => array(),
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Form/Type/MetricType.php]: []

**Comment by phaseinducer** [comment]
No `array()` -> `[]` conversion ?

**Comment by grena** [comment]
Not in our php-cs-fixer config :)
We progressively do conversion on master now

**Comment by phaseinducer** [comment]
Ok ! GTM then :+1:

**Pull request merged**